Source/WebCore/rendering/svg/RenderSVGModelObject.cpp

```diff
@@ -287,6 +287,11 @@ Path RenderSVGModelObject::computeClipPath(AffineTransform& transform) const
     return pathFromGraphicsElement(Ref { downcast<SVGGraphicsElement>(element()) });
 }
 
+void RenderSVGModelObject::paintSVGOutline(PaintInfo& paintInfo, const LayoutPoint& adjustedPaintOffset)
+{
+    paintOutline(paintInfo, LayoutRect(adjustedPaintOffset, repaintRectInLocalCoordinates().size()));
```

Comment:
It turns out visualOverflowRectEquivalent includes the outline, which is not what paintOutline expects. But I think we can use borderBoxRectEquivalent, since the element is already laid out and we also avoid (re)computing the bounding box.